libs/designsystem/shared/floating/src/floating.directive.spec.ts

```diff
@@ -70,6 +71,49 @@ describe('FloatingDirective', () => {
     expect(spectator.component).toBeTruthy();
   });
 
+  describe('automatic host position update behaviour', () => {
+    describe('floating element is not shown', () => {
+      it('should NOT peform position updates', fakeAsync(() => {
+        spyOn(directive, <never>'updateHostElementPosition');
```

Comment:
Consider adding the returned spy to a variable and reuse it in your expects to improve readability a little bit 
```suggestion
        const updateHostElementPositionSpy = spyOn(directive, <never>'updateHostElementPosition');
```

Reply:
Optionally you could do `const updateHostElementPositionSpy = spyOn<any>(directive, 'updateHostElementPosition');
` to match the style of the rest of the codebase

Reply:
Yes I should have checked for the general style.

I went with the `<never>` approach as it does not produce lint warnings, while `<any>` does